gsy_framework/sim_results/electric_blue/aggregate_results.py

```diff
@@ -103,6 +104,17 @@ def to_dict(self) -> Dict:
         """Generate a dictionary for saving the data into DB."""
         return {f.name: getattr(self, f.name) for f in fields(self)}
 
+    @classmethod
+    def from_dict(cls, device_stats_dict: Dict) -> "ForwardDeviceStats":
+        """Return ForwardDeviceStats from dict, with deserialized DateTimes
+        from YYYY-MM-DDTHH:mm format."""
+        if time_slot := device_stats_dict.get("time_slot"):
```

Comment:
Since gsy-framework is also used in gsy-e. Using of the Walrus operator will cause PyPy to panic. So, I guess we should remove it from here.

Reply:
I thought that if we use this module only in gsy-web we are fine. But I can change it if it's otherwise.

Reply:
I personally do not like this operator. Doing two things in on go is somehow I con not follow easily and I do not see the need here. What is the harm in doing the following? 
```suggestion
        time_slot = device_stats_dict.get("time_slot")
        if time_slot:
```

Reply:
Sorry, I did not want to dominate the discussion here. I see that you have already changed the code accordingly. So i guess that you are convinced? If you have a pro-argument of using the walrus operator here, please explain. Thanks a lot!

Reply:
Did not respond till now because it is a highly subjective matter (there are numerous flame wars online about the matter), however I am also with Hannes in this (and more specifically, I find that the walrus operator increases readability only in while loops). I might be overly biased by C++ where the assignment operator could be used in the same way as the Python walrus operator, but it was the source of a lot of hard-to-find-bugs, mainly because the control flow of the code was not evident to the developer on first glance.

Reply:
Well, I've just used that to make the code shorter, but in fact it made the code harder to read in such simple case. Agree, this operator is indeed useful in special cases, and probably should be used with caution. Thanks for the comments!